File: add/codesnippet/CSharp/f-system.net.ipaddress.i_2_1.cs
  // This method displays the value of the current host's Any address in  
  // standard compressed format. The Any address is used by the host to enable
  // listening to client activities on all the interfaces for a given port.
  private static void displayIPv6AnyAddress()
  {
    try
    {
      // Get the Any address.
      IPAddress any = IPAddress.IPv6Any;

      // Transform the Any address to a string using the overloaded
      // ToString() method. Note that the resulting string is in the compact 
      // form: "::".
      string ipv6Any = any.ToString();
      
      // Display the Any address.
      Console.WriteLine("The IPv6 Any address is: " + ipv6Any);
    }
    catch (Exception e) 
    {
      Console.WriteLine("[displayIPv6AnyAddress] Exception: " + e.ToString());
    }
  }

File: add/codesnippet/VisualBasic/f-system.net.ipaddress.i_2_1.vb
   ' This function prints the value of the current host's Any address in  
   ' standard compressed format. The Any address is used by the host to enable
   ' listening to client activities on all the interfaces for a given port.
   Private Shared Sub printIPv6AnyAddress()
      Try
         ' Get the Any address.
         Dim any As IPAddress = IPAddress.IPv6Any
         
         ' Transform the Any address to a string using the overladed
         ' ToString() method. Note that the resulting string is in the compact 
         ' form: "::".
         Dim ipv6Any As String = any.ToString()
         
         ' Display the Any address.
         Console.WriteLine(("The IPv6 Any address is: " + ipv6Any))
      Catch e As Exception
         Console.WriteLine(("[printIPv6AnyAddress] Exception: " + e.ToString()))
      End Try
   End Sub 'printIPv6AnyAddress
   